eox_audit_model/models.py

```diff
@@ -165,16 +168,22 @@ def execute_action(cls, action, method, parameters, notes=None):
             raise error
         finally:
             if getattr(settings, 'ALLOW_EOX_AUDIT_MODEL', True):
-                create_audit_register.delay(
-                    action=action,
-                    status=status,
-                    method_name=getattr(method, '__name__', 'Missing method name'),
-                    captured_logs='\n'.join(logs.formatted_records),
-                    traceback_log=traceback.format_exc(),
-                    input_parameters=parameters,
-                    output_parameters=result.__repr__(),
-                    notes=notes
-                )
+
+                audit_data = {
```

Comment:
Why not pass the arguments directly as before instead of creating the dict and unzipping it later?